pkg/api/status.go

```diff
@@ -25,8 +25,10 @@ const (
 )
 
 func NewErrorFields(status StatusCode, extraInfo string) *ErrorFields {
+	emptyString := ""
 	errFields := ErrorFields{
-		Status: int(status),
+		Status:    int(status),
+		ExtraInfo: &emptyString,
```

Comment:
You could use this instead of creating that empty string variable PtrTo(""),

Reply:
yeah, I tried that but `PtrTo` is undefined there and didn't want to copy function.